libs/inspecjs/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "inspecjs",
+  "version": "2.4.6",
+  "description": "© 2018 The MITRE Corporation.",
+  "files": [
+    "dist"
+  ],
+  "main": "./src/inspecjs.ts",
+  "dependencies": {},
+  "devDependencies": {
+    "@types/node": "^12.20.7",
+    "@typescript-eslint/eslint-plugin": "^2.34.0",
+    "@typescript-eslint/parser": "^2.34.0",
+    "eslint": "^6.8.0",
+    "eslint-config-prettier": "^6.15.0",
+    "eslint-plugin-prettier": "^3.3.1",
+    "prettier": "^1.19.1",
+    "typedoc": "^0.20.35",
+    "typescript": "^3.9.9"
+  },
+  "scripts": {
+    "build": "rm -r ./dist; tsc",
```

Comment:
Instead of using `rm` here, this should instead just use a prebuild step just like heimdall-lite does: https://github.com/mitre/heimdall2/blob/master/apps/frontend/package.json#L16